.gitlab-ci.yml

```diff
@@ -3,6 +3,7 @@ stages:
   - build_static_assets
   - build_docker_image
   - deploy
+  - lighthouse
```

Comment:
If lighthouse fails I'm assuming we don't want to deploy? If so, `lighthouse` wants to go above the `deploy` stage

Reply:
So for what I said in https://github.com/ebi-uniprot/uniprot-website/pull/356#discussion_r623153110 I'd say:
```
  - test (<- include lighthouse_subset, all commits)
  - build_static_assets
  - build_docker_image
  - deploy
  - lighthouse (<- lighthouse full on the deployed version, only on main)
```